README.md

```diff
@@ -2,64 +2,107 @@
-   exposes infrastructure via an API.
-    - Examples: [Google Cloud
-      Platform](https://github.com/crossplane/provider-gcp), [Amazon Web
-      Services](https://github.com/crossplane/provider-aws),
-      [Azure](https://github.com/crossplane/provider-azure),
-      [Alibaba](https://github.com/crossplane/provider-alibaba),
-      [Github](https://github.com/crossplane/provider-github)
-3. **Stacks**: a bundled set of custom resources that together represent an
-   environment on a cloud provider. The bundle of instances can be created by a
-   single custom resource.
-   - Examples: [GCP Sample
-     Stack](https://github.com/crossplane/stack-gcp-sample), [AWS Sample
-     Stack](https://github.com/crossplane/stack-aws-sample), [Azure Sample
-     Stack](https://github.com/crossplane/stack-azure-sample)
-4. **Applications**: a deployable unit of code and configuration, which, when
-   created, may involve provisioning new services which are managed by a
-   `provider`, or consuming services created by a `stack`.
-    - Examples: [Wordpress](https://github.com/crossplane/app-wordpress)
-
-The full vision and architecture of the Crossplane project is described in our
-[architecture document].
-
-For more information, take a look at the official Crossplane [documentation].
+Crossplane is an open source Kubernetes add-on that extends any cluster with
+the ability to provision and manage cloud infrastructure, services, and
+applications using kubectl, GitOps, or any tool that works with the Kubernetes
+API.
+
+With Crossplane you can:
+
+* **Provision & manage cloud infrastructure and services with kubectl**
+  * [Install Crossplane] to provision and manage cloud infrastructure and
+    services from any Kubernetes cluster.
+  * Provision infrastructure primitives from any provider (GCP, AWS, Azure,
+    Alibaba, on-prem) and use them alongside existing application configurations.
+  * Version, manage, and deploy with your favorite tools and existing
+    workflows that you’re using with your clusters today.
+
+* **Publish custom infrastructure resources for your applications to use**
+  * Define, compose, and publish your own [infrastructure resources] with
+    declarative YAML, resulting in your own infrastructure CRDs being added to
+    the Kubernetes API for applications to use.
+  * Hide infrastructure complexity and include policy guardrails, so
+    applications can easily and safely consume the infrastructure they need,
+    using any tool that works with the Kubernetes API.
+  * Consume infrastructure resources alongside any Kubernetes application to
+    provision and manage the cloud services they need with Crossplane as an
+    add-on to any Kubernetes cluster.
+
+* **Deploy applications using a team-centric approach with OAM**
+  * Define cloud native applications and the infrastructure they require with
+    the Open Application Model
+    ([OAM](https://github.com/oam-dev/spec/releases/tag/v1.0.0-alpha.2)).
```

Comment:
The master branch of OAM spec is v1alpha2 and will be upgraded to link Crossplane as implementation, it's very safe to link https://github.com/oam-dev/spec instead of a release page which lacks documentation and sample.

At meantime, I will add a "current release" section on OAM spec README so it would be more reliable.

Reply:
I'm a little wary of linking to master of the oam spec repo. The OAM spec is constantly evolving quite rapidly, and we can't always guarantee that master of Crossplane will match master of the OAM spec. It seems safer to me to pin to the specific version of the spec that Crossplane currently supports.

Reply:
That's a good point. Let's leave it pinned to the specific version Crossplane supports for now.